release/build_release.sh

```diff
@@ -73,6 +73,39 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_BUILD_DIR}"
 }
 
+function cherry_pick_commits {
+	local liferay_release_tickets_array
+
+	IFS=',' read -ra liferay_release_tickets_array <<< "${LIFERAY_RELEASE_TICKETS}"
+
+	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
+
+	git pull origin -q "${LIFERAY_RELEASE_GIT_REF}"
+
+	for liferay_release_ticket in "${liferay_release_tickets_array[@]}"
+	do
+		git checkout -q master
+
+		local liferay_release_commits_sha=($(git log --grep="${liferay_release_ticket}" --pretty=format:%H --reverse))
+
+		git checkout -q "${LIFERAY_RELEASE_GIT_REF}"
+
+		for liferay_release_commit_sha in "${liferay_release_commits_sha[@]}"
+		do
+			git cherry-pick --strategy-option theirs "${liferay_release_commit_sha}" > /dev/null
+
+			if [ $? -eq 0 ]
```

Comment:
Please see https://github.com/natocesarrego/liferay-docker/pull/6#discussion_r1572330130

Reply:
Is not needed.